src/tracing/index.ts

```diff
@@ -30,7 +31,19 @@ Sentry.init({
       startTransactionOnPageLoad: true,
     }),
   ],
-  beforeSend: filterKnownErrors,
+  beforeSend(event: ErrorEvent, hint: EventHint) {
+    /**
```

Comment:
This is a comment, not a doc-comment, so use `/*` not `/**`:

```suggestion
    /*
```